src/app/components/Header/Header.tsx

```diff
@@ -1,14 +1,89 @@
 "use client";
+import { useState } from "react";
 import { styles } from "./Header.style";
-import { Box, Typography } from "@mui/material";
-import { HeaderProps } from "@/app/general/interfaces";
+import { Box, Typography, Button } from "@mui/material";
+import Table from "@/app/components/Table";
+import Dialog from "@/app/components/Dialog";
+import {
+    HeaderProps,
+    NameAndDescription,
+    generalObject,
+} from "@/app/general/interfaces";
+import { strOrNum } from "@/app/general/types";
+
+export default function Header({ bot }: HeaderProps) {
+    const [openAttribute, setOpenAttribute] = useState<boolean>(false);
+    const [openData, setOpenData] = useState<boolean>(false);
+
+    const botHeaders = bot?._data.headers;
+    const botColumns = bot?._data.columns;
+
+    const rows: generalObject<strOrNum>[] = [];
+
+    [0, 1, 2, 3, 4].forEach((index) => {
```

Comment:
Something is not elegant here? Why not iterate the column instead of [0...5]?

Reply:
fixed